### DISCUSSION TITLE
실행 컨텍스트 딥다이브 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#8] [Announcements]
# 실행 컨텍스트 딥다이브 | 정현수 기술 블로그

딥다이브 실행 컨텍스트 파트를 읽고

https://junghyeonsu-dev.vercel.app/posts/Execution-context#comments-container

**Comment by o-bard-o** [comment]
굉장히 자세히 정리해 주셨네요 .. 감사합니다 !
